Dockerfile

```diff
@@ -2,19 +2,19 @@ ARG ruby_version=3.3
 ARG base_image=ghcr.io/alphagov/govuk-ruby-base:$ruby_version
 ARG builder_image=ghcr.io/alphagov/govuk-ruby-builder:$ruby_version
 
-FROM $builder_image AS builder
+FROM --platform=$TARGETPLATFORM $builder_image AS builder
 
 WORKDIR $APP_HOME
 COPY Gemfile* .ruby-version ./
-RUN bundle install
+RUN bundle install --jobs=$(nproc)
```

Comment:
optional: this should have little effect cos the builder image [already sets](https://github.com/alphagov/govuk-ruby-images/blob/8181cf8/builder.Dockerfile#L9) `BUNDLE_JOBS`.

Reply:
OTOH if what you want is to reduce the concurrency to the number of cores on the runner then this does what you want. I'd be inclined to leave it alone if you haven't benchmarked it though, otherwise it's hard to know whether it's an improvement or a regression.

Reply:
`bundle install` is often (but not always) I/O-bound btw.